idc_index/index.py

```diff
@@ -623,19 +624,23 @@ def _validate_update_manifest_and_get_download_size(
 
         endpoint_to_use = None
 
-        if validate_manifest:
-            # Check if crdc_instance_uuid is found in the index
-            if not all(merged_df["crdc_series_uuid_match"]):
-                missing_manifest_cp_cmds = merged_df.loc[
-                    ~merged_df["crdc_series_uuid_match"], "manifest_cp_cmd"
-                ]
-                missing_manifest_cp_cmds_str = f"The following manifest copy commands do not have any associated series in the index: {missing_manifest_cp_cmds.tolist()}"
-                raise ValueError(missing_manifest_cp_cmds_str)
+        # Check if any crdc_series_uuid are not found in the index
+        if not merged_df["crdc_series_uuid_match"].all():
```

Comment:
fyi

![image](https://github.com/user-attachments/assets/949e6839-3622-40cd-a01c-7cfd6cb6c144)

Reply:
I see. I was experimenting with alternative ways to fix the bug alluded to in the above, which did not work. I will change back to built-in.